ngraph/core/include/ngraph/node.hpp

```diff
@@ -233,7 +236,12 @@ namespace ngraph
         /// Sets the number of outputs
         void set_output_size(size_t output_size);
 
-        void revalidate_and_infer_types() { validate_and_infer_types(); }
+        void invalidate_values();
```

Comment:
Can we make this method private?

Reply:
nope

Reply:
I didn't see any external usage of this method. It is used only inside revalidate_and_infer_types.